deployment/src/main/java/io/quarkiverse/jasperreports/deployment/JasperReportsProcessor.java

```diff
@@ -430,62 +423,64 @@ void watchReportFiles(BuildProducer<HotDeploymentWatchedFileBuildItem> watchedPa
 
     @BuildStep
     void compileReports(ReportConfig config, List<ReportFileBuildItem> reportFiles,
-            BuildProducer<GeneratedClassBuildItem> compiledReportProducer, OutputTargetBuildItem outputTarget) {
+            BuildProducer<GeneratedClassBuildItem> compiledReportProducer,
+            BuildProducer<CompiledReportFileBuildItem> compiledReportFileProducer,
+            OutputTargetBuildItem outputTarget) {
 
-        if (config.build().enable()) {
-            Log.debugf("Found %s report(s) to compile?", reportFiles.size());
-            try {
-                // make sure the destination path exists before we try to write files to it
-                Path outputFilePath = Path.of(outputTarget.getOutputDirectory().toString(), "classes",
-                        config.build().destination().get().toString());
+        if (!config.build().enable()) {
+            Log.debug("Automatic report compilation disabled");
+        }
 
-                if (!Files.exists(outputFilePath)) {
-                    Files.createDirectories(outputFilePath);
-                }
+        Log.debugf("Found %s report(s) to compile?", reportFiles.size());
```

Comment:
Rats, I forgot to remove that question mark.  Could you remove that?

Reply:
Yep I got it!